src/platform/smart-home-app.ts

```diff
@@ -3,13 +3,13 @@
  */
 /// <reference types="@google/local-home-sdk" />
 
-import { MockLocalHomePlatform } from './mock-local-home-platform';
+import {MockLocalHomePlatform} from './mock-local-home-platform';
 
 export class AppStub implements smarthome.App {
   private version: string;
-  public identifyHandler: smarthome.IntentFlow.IdentifyHandler;
-  public executeHandler: smarthome.IntentFlow.ExecuteHandler;
-  public reachableDevicesHandler: smarthome.IntentFlow.ReachableDevicesHandler;
+  public identifyHandler!: smarthome.IntentFlow.IdentifyHandler;
```

Comment:
Does this need the `!` syntax?

Reply:
Added `| undefined` to these types.  Also went ahead and added null checks which throw errors in the `IdentifyRequest` logic (which will be moved to another class in #32, but I figure is worth adding here just to keep master clean).